pkg/search/controller.go

```diff
@@ -188,6 +188,12 @@ func (c *Controller) doCacheCluster(cluster string) error {
 		return nil
 	}
 
+	if !util.IsClusterReady(&cls.Status) {
+		klog.Warningf("cluster %s is notReady try to stop this cluster informer", cluster)
+		c.InformerManager.Stop(cluster)
```

Comment:
If the cluster gets ready again, all the informers will be recreated?

Reply:
klog.Warningf("cluster %s is notReady try to stop this cluster informer", cluster.Name) ?

Reply:
> If the cluster gets ready again, all the informers will be recreated?
when cluster member1 ready again, only member1 be recreated
![image](https://user-images.githubusercontent.com/89568107/210031165-9ab44184-3400-4107-858f-942d7c85e628.png)

Reply:
> klog.Warningf("cluster %s is notReady try to stop this cluster informer", cluster.Name) ?

this cluster type is string

Reply:
> If the cluster gets ready again, all the informers will be recreated?

It seems will be recreated. When cluster status updates, it will execute  `doCacheCluster` once.